packages/react-composites/src/composites/CallComposite/selectors/localAndRemotePIPSelector.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { _isInCall, _isPreviewOn, _videoGalleryRemoteParticipantsMemo } from '@internal/calling-component-bindings';
+import { RemoteParticipantState } from '@internal/calling-stateful-client';
+import { LocalVideoStreamState } from '@internal/calling-stateful-client';
+import * as reselect from 'reselect';
+import {
+  getDeviceManager,
+  getDisplayName,
+  getDominantSpeakers,
+  getLocalVideoStreams,
+  getRemoteParticipants
+} from './baseSelectors';
+import { callStatusSelector } from './callStatusSelector';
+
+/**
+ * Picture in picture in picture needs to display the most-dominant remote speaker, as well as the local participant video.
+ * @private
+ */
+export const localAndRemotePIPSelector = reselect.createSelector(
+  [
+    getDisplayName,
+    callStatusSelector,
+    getDeviceManager,
+    getLocalVideoStreams,
+    getRemoteParticipants,
+    getDominantSpeakers
+  ],
+  (displayName, callStatus, deviceManager, localVideoStreams, remoteParticipants, dominantSpeakers) => {
+    // Get local video stream details
+    let localVideoStream: LocalVideoStreamState | undefined;
```

Comment:
I'd extract a `localVideoStream` selector. Couple with `mostDominantSpeaker` selector, it'll allow you to short-circuit calaculations when only the local vs remote participants change.

Reply:
Updated to seperate selectors - can you take a look and make sure it was done correctly